Assets/Scripts/Canvas/PropertyContainer/PointContainer.cs

```diff
@@ -28,39 +29,32 @@ public void Start()
     public void SetNode(Node node)
     {
       _node = node;
-      
+
       (x.text, y.text, z.text) = _node.Position.ToStringTuple();
       (theta.text, phi.text, psi.text) = _node.Rotation.ToStringTuple();
     }
 
-    public bool IsInputCorrect()
-    {
-      return float.TryParse(x.text, out var x_res) && float.TryParse(y.text, out var y_res)
-        && float.TryParse(z.text, out var z_res) && float.TryParse(theta.text, out var theta_res)
-        && float.TryParse(phi.text, out var phi_res) && float.TryParse(psi.text, out var psi_res);
-    }
+    public bool IsInputCorrect() =>
+      float.TryParse(x.text, out var x_res) && float.TryParse(y.text, out var y_res)
+                                            && float.TryParse(z.text, out var z_res) &&
+                                            float.TryParse(theta.text, out var theta_res)
+                                            && float.TryParse(phi.text, out var phi_res) &&
+                                            float.TryParse(psi.text, out var psi_res);
```

Comment:
That formatting looks inconsistent.
Also, I think my change here got lost in merges somewhere. The out values are not used so we can use wildcard out _ instead. Consider doing this again.